File: JAVApb5/src/main/java/concurs/repository/IRepositoryProba.java
package concurs.repository;

import concurs.domain.Proba;

import java.util.List;

public interface IRepositoryProba {
    public Proba add(Proba e);
    public Proba delete(Proba e);
    public Proba update(Proba e);
    public Proba findOne(Long id);
    public List<Proba> findAll();
}

File: CSharp/CSharp/CSharp/repository/IRepositoryProba.cs
using CSharp.domain;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace CSharp.repository
{
    interface IRepositoryProba
    {
        Proba Add(Proba e);
        Proba Delete(Proba e);
        Proba Update(Proba e);
        Proba FindOne(long id);
        List<Proba> FindAll();
    }
}
